scanpy_scripts/cmd_options.py

```diff
@@ -533,6 +533,13 @@
             'in the format of "-p name min max". '
             'Multiple -p entries allowed.',
         ),
+        click.option(
+            '--boolean', '-b',
+            type=(click.STRING),
+            multiple=True,
+            help='A boolean variable to filter by. Multiple -b entries '
+            'allowed.'
+        ),
```

Comment:
I am not sure we need a dedicated boolean option. The reason that `--mito False` doesn't work might be that "False" is not interpreted as boolean. A simple conversion of all categorical/boolean attributes to string might be what it takes for it to work.

Reply:
I checked: Boolean columns are ignored. I'll verify tomorrow.